apps/src/netsim/NetSimRouterNode.js

```diff
@@ -1051,11 +1116,53 @@ NetSimRouterNode.prototype.updateRouterQueue_ = function (rows) {
   }
 
   this.routerQueueCache_ = newQueue;
-  // TODO: Drop packets that exceed queue memory (IF not already processing?)
   this.recalculateSchedule();
+  this.enforceMemoryLimit_();
+
   // Propagate notification of queue change (for stats, etc)
 };
 
+NetSimRouterNode.prototype.enforceMemoryLimit_ = function () {
+  if (this.enforcingMemoryLimit_) {
```

Comment:
Maybe currentlyEnforcingMemoryLimit_? I originally read this expecting this to be a sort of static option instead of a state indicator.